Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -105,18 +115,26 @@ int main()
         // notifications to the sample
         case ExtendedActivationKind::Launch:
         {
-            // Request a WNS Channel URI which can be passed off to an external app to send notifications to.
-            // The WNS Channel URI uniquely identifies this app for this user and device.
-            PushNotificationChannel channel{ RequestChannel() };
-            if (!channel)
+            if (pushNotificationManager.IsSupported())
             {
-                std::cout << "\nThere was an error obtaining the WNS Channel URI" << std::endl;
-
-                if (remoteId == winrt::guid { "00000000-0000-0000-0000-000000000000" })
+                // Request a WNS Channel URI which can be passed off to an external app to send notifications to.
+                // The WNS Channel URI uniquely identifies this app for this user and device.
+                PushNotificationChannel channel{ RequestChannel() };
+                if (!channel)
                 {
-                    std::cout << "\nThe remoteId has not been set. Refer to the readme file accompanying this sample\nfor the instructions on how to obtain and setup a remote id" << std::endl;
+                    std::cout << "\nThere was an error obtaining the WNS Channel URI" << std::endl;
+
+                    if (remoteId == winrt::guid{ "00000000-0000-0000-0000-000000000000" })
```

Comment:
Having this check here from a samples perspective is incorrect. It needs to be done before we request a channel.

Reply:
I'm thinking about it from the perspective of the developer copy pasting this pattern or code : )

Reply:
Besides, if the remoteID is GUID_NULL, I think the Channel request API would throw.

Reply:
That's for all the devs who are going to download the sample, open the solution in VS and hit F5 without having read the docs. Instead of the sample just failing with a cryptic error, it reminds them to RTFM and update the remoteID.

Reply:
Ok. I wont nit on this then :). Its a minor issue and shouldnt block. Signing off.